Tests/XCUITests/IntegrationTests.swift

```diff
@@ -48,14 +48,15 @@ class IntegrationTests: BaseTestCase {
             return true
         }
         app.swipeDown()
+        sleep(5)
```

Comment:
Is this sleep here necessary? Cant a wait function be used here instead? Just asking, if this is the only solution then i am ok with it?

Reply:
Let me remove it for now and open a separate PR to address the issue.